dist/filter-menu/ds4/filter-menu.css

```diff
@@ -0,0 +1,154 @@
+span.filter-menu-aria,
+span.filter-menu-form {
+  background-color: #fff;
+  border: 1px solid #ccc;
+  border-radius: 16px;
+  display: inline-block;
+  max-width: 288px;
```

Comment:
Hmm, not sure if we should have a max width on the non-button version...?

Reply:
This is in the spec.

Reply:
I see what you mean. I will leave note about moving it. The `filter-menu-button` will be a fast follow and I can complete it's removal then.